cli/src/commands/restore.rs

```diff
@@ -70,6 +70,9 @@ pub(crate) struct RestoreArgs {
     /// the user might not even realize something went wrong.
     #[arg(long, short, hide = true)]
     revision: Option<RevisionArg>,
+    /// Do not modify the content of the children of the rewritten commits
```

Comment:
nit: I slightly prefer the doc string of `diffedit`.

"Preserve the content (not the diff) when rebasing descendants
...
"